run-x/opta
# Add wait option to helm-chart

# Description
- I also fixed the trailing '.' problem in gcp dns

# Test plan
- Use wait=false and atomic=false to get airflow helm chart to work
- Verify apply on older environments - to make sure the `.` change isn't backwards incompatible
